### ISSUE TITLE
Name parser not using show's title

### ISSUE THREAD

**Issue by duramato** [pymedusa/Medusa#3676]
```
2018-01-27 14:57:23 INFO     SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] Performing season pack search for Swedish Dicks
2018-01-27 14:57:23 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] Search mode: Season
2018-01-27 14:57:23 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] Search string: {u'category': u'Season', u'series': u'Swedish Dicks', u'name': u'Season 2'}
2018-01-27 14:57:24 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] Found result: Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL with 25 seeders and 1 leechers
2018-01-27 14:57:24 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] Found result: Swedish.Dicks.S02.720p.WEBRip.DD2.0.x264-StB with 20 seeders and 0 leechers
2018-01-27 14:57:24 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] Found 2 unique items
2018-01-27 14:57:24 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] Found qualities: [64, 32]
2018-01-27 14:57:24 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] No unknown qualities in results
2018-01-27 14:57:24 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] Error during parsing of release name: Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL, with error: Unable to match Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL to a series in your database. Parser result: language: sv, title: Dicks, season: 2, screen_size: 1080p, format: WEB-DL, audio_codec: AC3, audio_channels: 5.1, video_codec: h264, release_group: DBRETAiL, type: episode, parsing_time: 0.243055105209, total_time: 0.661642074585, absolute_episode: [], quality: 1080p WEB-DL, episode: []
2018-01-27 14:57:25 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] Error during parsing of release name: Swedish.Dicks.S02.720p.WEBRip.DD2.0.x264-StB, with error: Unable to match Swedish.Dicks.S02.720p.WEBRip.DD2.0.x264-StB to a series in your database. Parser result: language: sv, title: Dicks, season: 2, screen_size: 720p, format: WEBRip, audio_codec: AC3, audio_channels: 2.0, video_codec: h264, video_encoder: x264, release_group: StB, type: episode, parsing_time: 0.210155963898, total_time: 0.628087997437, absolute_episode: [], quality: 720p WEB-DL, episode: []
2018-01-27 14:57:25 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] We aren't interested in this result: Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL with url: 
2018-01-27 14:57:25 DEBUG    SEARCHQUEUE-MANUAL-67642 :: [BTN] :: [b59449d] We aren't interested in this result: Swedish.Dicks.S02.720p.WEBRip.DD2.0.x264-StB with url: 
```

The show title is `Swedish Dicks`, but it doesn't parse the show name correctly and parses the title as a language, but _afaik_ guessit uses the show's titles.

```
# guessit: 2.1.4  rebulk: 0.9.0
? Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL
: language: sv
  title: Dicks
  season: 2
  screen_size: 1080p
  format: WEB-DL
  audio_codec: AC3
  audio_channels: 5.1
  video_codec: h264
  release_group: DBRETAiL
  type: episode
  parsing_time: 0.300627946854
```

This seems like an issue with our own version of guessit:

```
# guessit: 2.1.4  rebulk: 0.9.0
# show list: ['-T', 'Swedish Dicks']
? Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL
  language: sv
  title: Dicks
  season: 2
  screen_size: 1080p
  format: WEB-DL
  audio_codec: AC3
  audio_channels: 5.1
  video_codec: h264
  release_group: DBRETAiL
  type: episode
  parsing_time: 0.28922700882
```
```
guessit "Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL" -T "Swedish Dicks"
For: Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL
GuessIt found: {
    "title": "Swedish Dicks",
    "season": 2,
    "screen_size": "1080p",
    "format": "WEB-DL",
    "audio_codec": "AC3",
    "audio_channels": "5.1",
    "video_codec": "h264",
    "release_group": "DBRETAiL",
    "type": "episode"
}
```
@ratoaq2

**Comment by fernandog** [comment]
Use report_guessit command line. It uses Medusa's rules

**Comment by duramato** [comment]
Thats the first output on the OP, right after the medusa logs.

**Comment by sharkykh** [comment]
Looks like it's fixed in guessit 3.0.0.
So will be fixed when #4244 is done.
```yml
$ python tests/report_guessit.py "Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL" "Swedish Dicks"
# guessit: 3.0.0  rebulk: 0.9.0
# show list: ['Swedish Dicks']
? Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL
  title: Swedish Dicks
  season: 2
  screen_size: 1080p
  source: Web
  audio_codec: AC3
  audio_channels: 5.1
  video_codec: H.264
  release_group: DBRETAiL
  type: episode
  parsing_time: 0.0880000591278

$ guessit -t episode "Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL"
For: Swedish.Dicks.S02.1080p.WEB-DL.DD5.1.H.264-DBRETAiL
GuessIt found: {
    "title": "Swedish Dicks",
    "season": 2,
    "screen_size": "1080p",
    "source": "Web",
    "audio_codec": "Dolby Digital",
    "audio_channels": "5.1",
    "video_codec": "H.264",
    "release_group": "DBRETAiL",
    "type": "episode"
}
```
